src/commands/banner.ts

```diff
@@ -0,0 +1,28 @@
+import { ChatInputCommandInteraction, SlashCommandBuilder } from 'discord.js';
+import { CustomClient } from '../typings/Extensions.js';
+
+export const name = 'banner';
+export const data = new SlashCommandBuilder()
+  .setName(name)
```

Comment:
for all commands, remove the "name" var. the data var already has "name" becuase SlashCommandBuilder outputs a JSON and then can be called on data.name

Reply:
Fixing in a commit